File: features/step_definitions/login_steps.rb
Given /^I am logged in as "(.*?)"$/ do |email|
  login(email, 'complex_password')
end

When /^I login as "(.*?)"$/ do |email|
  login(email, 'complex_password')
end

Then /^I should be logged in$/ do
  page.should have_css "body.dashboard.show"
end

Then /^I should not be logged in$/ do
  page.should have_content "Invalid"
end

When /^I login as "(.*?)" with an incorrect password$/ do |email|
  login(email, 'wrong_password')
end

When /^I click on the login button$/ do
  first('.login').click
end

When(/^I log in$/) do
  fill_in 'user_email', with: @user.email
  fill_in 'user_password', :with => 'complex_password'
  click_on 'sign-in-btn'
end

Then(/^I should see the group page$/) do
  page.should have_css('body.groups.show')
end

File: features/step_definitions/login_steps.r
Given /^I am a registered User$/ do
  User.create(:email => "fluffy@example.com", :password =>"password", name=>"Fluffy")
  end

When /^I am on the login page$/ do
   within("#index") do
   fill_in 'Login', :with => 'user@example.com'
   fill_in 'Password', :with => 'password'
  end
end

When /^I enter my details$/ do
  pending # express the regexp above with the code you wish you had
end

When /^click submit$/ do
  pending # express the regexp above with the code you wish you had
end

Then /^I am logged in$/ do
  pending # express the regexp above with the code you wish you had
end

